scripts/setup/setup-reportstream.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+[ -z "${CDCTI_HOME}" ] && echo "Error: Environment variable CDCTI_HOME is not set. Please refer to /scripts/README.md for instructions" && exit 1
+source "$CDCTI_HOME/scripts/lib/common.sh"
+
+# Check requirements for this script
+check_env_vars RS_HOME
+check_installed_commands yq
+
+# Need to CD to the RS codebase local working copy to run the gradlew commands
+CURRENT_DIR=$(pwd)
+cd "$RS_HOME" || exit
+
+# source "./prime-router/.vault/env/.env.local"
```

Comment:
Not sure what this comment means

Reply:
Missed that. We don't need it so will remove it